tests/testthat/test-independent-testing-meta_split.R

```diff
@@ -0,0 +1,47 @@
+library(dplyr)
+
+test_that("variable 'RACE' checking", {
+ expect_equal(names(meta_dummy() |> meta_split("RACE")), sort(unique(meta_dummy()$data_population$RACE)))
```

Comment:
I don't think the above test is what expected because it only tests the name of `"RACE"` match. And the split functionality is un-tested.

Reply:
This step is to check meta_split howmany race groups are creating